research/gaia/gremlin/gs_gremlin/src/graph_proxy/util.rs

```diff
@@ -0,0 +1,74 @@
+//
```

Comment:
Have no idea why you cannot directly import these functions from Gaia. You can import it into somewhere in graph_proxy for internal use. If it is private in Gaia, make it public is okay.

Reply:
Done